packages/react-ui/components/MobileLayout/MobileLayout.tsx

```diff
@@ -0,0 +1,165 @@
+      }
+
+      if (super.componentDidMount) {
+        super.componentDidMount();
+      }
+    }
+
+    componentWillUnmount() {
+      this.unsubscribeMediaQueries();
+
+      if (super.componentWillUnmount) {
+        super.componentWillUnmount();
+      }
+    }
+
+    public prepareMediaQueries = () => {
+      if (!this.theme) {
+        return;
+      }
+
+      this.layouts = this.getCreatedLayouts(this.theme);
+
+      this.currentMediaQeries = this.layouts.map((layout) => window.matchMedia(layout.mediaQuery));
+      this.currentMediaQeries.forEach((mql) => {
+        if (mql && mql.addEventListener) {
+          mql.addEventListener('change', this.checkLayoutsMediaQueries);
+        }
+      });
+
+      this.checkLayoutsMediaQueries();
+    };
+
+    public unsubscribeMediaQueries = () => {
+      if (!this.theme) {
+        return;
+      }
+
+      if (this.currentMediaQeries) {
+        this.currentMediaQeries.forEach((mql) => {
+          if (mql && mql.removeEventListener) {
+            mql.removeEventListener('change', this.checkLayoutsMediaQueries);
+          }
+        });
+      }
+    };
+
+    public checkLayoutsMediaQueries = (e?: MediaQueryListEvent) => {
+      if (!this.theme) {
+        return;
+      }
+
+      const matchedLayouts = this.layouts.filter((layout) => this.checkMQ(layout.mediaQuery));
+      const firstMatchedLayout: LayoutData | undefined = matchedLayouts[0];
+
+      if (e && e.matches && firstMatchedLayout && firstMatchedLayout.mediaQuery === e.media) {
+        if (this.state.layout !== firstMatchedLayout.name) {
+          this.setState({
+            layout: firstMatchedLayout.name,
+          });
+        }
+
+        return;
+      }
+
+      if (firstMatchedLayout && this.state.layout !== firstMatchedLayout.name) {
+        this.setState({
+          layout: firstMatchedLayout.name,
+        });
+
+        return;
+      }
+
+      if (!firstMatchedLayout && this.state.layout !== LayoutMode.Desktop) {
+        this.setState({
+          layout: LayoutMode.Desktop,
+        });
+      }
+    };
+
+    public createLayoutData(name: LayoutMode, mediaQuery: string): LayoutData {
+      return {
+        name,
+        mediaQuery,
+      };
+    }
+
+    public getCreatedLayouts(theme: Theme) {
+      const layouts: LayoutData[] = [];
+
+      try {
+        Object.values(LayoutMode).forEach((layout) => {
+          // @ts-ignore
+          const layoutThemeMediaQuery: string | undefined = theme[LayoutMQThemeKeys[layout]];
+          if (layoutThemeMediaQuery) {
+            layouts.push(this.createLayoutData(layout, layoutThemeMediaQuery));
+          }
+        });
+      } catch (error) {
+        return layouts;
+      }
+
+      return layouts;
+    }
+
+    public checkMQ(mediaQuery: string) {
+      if (canUseDOM) {
+        return window.matchMedia(mediaQuery).matches;
+      }
+    }
+
+    public render(): JSX.Element {
+      return <>{super.render()}</>;
+    }
```

Comment:
Метод `render()` возвращает тип [`ReactNode`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/v16/index.d.ts#L490)
Проще просто избавиться от типа, и от фрагмента.
```suggestion
    public render() {
      return super.render();
    }
```